packages/crayons-core/src/components/inline-message/inline-message.tsx

```diff
@@ -0,0 +1,157 @@
+import {
+  Component,
+  Prop,
+  h,
+  Host,
+  Watch,
+  Element,
+  Method,
+  EventEmitter,
+  Event,
+} from '@stencil/core';
+
+const iconMap = {
+  error: 'alert',
+  warning: 'info',
+  info: 'info',
+  success: 'success',
+};
+
+const iconColorMap = {
+  error: '#e43538',
+  warning: '#c7502f',
+  info: '#264966',
+  success: '#00795b',
+};
+
+@Component({
+  tag: 'fw-inline-message',
+  styleUrl: 'inline-message.scss',
+  shadow: true,
+})
+export class InlineMessage {
+  @Element() host!: HTMLElement;
+
+  private autoHideTimeout;
+
+  /**
+   * Makes the inline message closable.
+   */
+  @Prop() closable = false;
+
+  /**
+   * The type of inline message to be displayed. Defaults to info.
+   */
+  @Prop() type: 'success' | 'warning' | 'info' | 'error' = 'info';
+
+  /**
+   * The duration in milliseconds for which inline message will be shown.
+   */
+  @Prop() duration = Infinity;
+
+  /**
+   * Indicates whether the inline message is open or not.
+   */
+  @Prop({ mutable: true, reflect: true }) open = false;
+
+  /**
+   * Triggered when inline message is shown.
+   */
+  @Event() fwShow!: EventEmitter;
+
+  /**
+   * Triggered when inline message is hidden.
+   */
+  @Event() fwHide!: EventEmitter;
+
+  startAutoHide() {
+    clearTimeout(this.autoHideTimeout);
+    if (this.open && this.duration < Infinity) {
+      this.autoHideTimeout = setTimeout(() => this.hide(), this.duration);
+    }
+  }
+
+  @Watch('open')
```

Comment:
For some reason, I am not able to find the open prop! where is it defined?

Reply:
It is defined on line 55